views/partials/inventory/product-order/search-grid.jade

```diff
@@ -8,11 +8,8 @@
             span.item-text.hidden-xs Cancel
         .h4
           strong
-            a(ng-if="searchVM.role === 'Customer'", ui-sref="inventory.productOrder.detail({ productOrderId: productOrder.id })", ui-sref-opts="{ inherit: false }")
+            a(ui-sref="inventory.productOrder.detail({ productOrderId: productOrder.id, role: searchVM.role })", ui-sref-opts="{ inherit: false }")
```

Comment:
Because I thought that the user's role was to be required into the productOrder detailView but was not. I forgot to delete it.